### ISSUE TITLE
Extend e2e test coverage for ScaledJob

### ISSUE THREAD

**Issue by zroubalik** [kedacore/keda#3661]
### Proposal

Cover advanced options/scaling strategies of ScaledJob in e2e tests.

**Comment by geoffrey1330** [comment]
Hi @JorTurFer could you please assign this issue to me
cc @zroubalik

**Comment by jansworld** [comment]
Hey @JorTurFer is this still active? If so, I wouldn't mind taking a crack at it as a first contribution

**Comment by JorTurFer** [comment]
yes! if you are willing to improve the coverage por ScaledJobs, it's a quite welcomed contribution ❤️ 

I've assigned it to you, thanks!
